squash/supervisord.conf

```diff
@@ -0,0 +1,26 @@
+[supervisord]
+nodaemon=true                                 ; run supervisor in foreground otherwise docker run exits
+
+[program:mysql]
+user=root
+command=/usr/bin/mysqld_safe
+stdout_logfile=/var/log/supervisor/mysql.log
+redirect_stderr=true                          ; Save stderr in the same log
+autostart=true
+
+[program:django]
+user=root
```

Comment:
It is generally a good idea not to run any services as the root user, even in a container.